tofuplot.py

```diff
@@ -143,6 +144,14 @@ def _str2bool(v):
                         nargs='+', default=None)
     parser.add_argument('-t0', '--t0', type=str, required=False,
                         help='Reference time event setting t = 0', default=_T0)
+    parser.add_argument('-t', '--t', type=float, required=False,
+                        help='Input time when needed')
```

Comment:
@Didou09 seems like replacing renaming a flag that you used to be used for something else, and adding a new one  (type) for the old one (tokamak), can lead to confusion. Could the long flag for the time could be at least `--time`, or even better `--input-time`?

Reply:
Discussed together, name flag -tok (short version) is actually probably better for --tokamak and -t for time is also more intuitive.
So it's fine for me.

The question on the long version (i.e.: --t vs --time) remains, I'm not sure what to think, because updating to --time would also require updating the corresponding keyword argument of the python function.

I suggest : leave it as is (--t) for now, and we'll see later if users mention any issue / ambiguity with that

Reply:
fine by me :)